concepticondata/conceptlists/Capell-1956-558.tsv

```diff
@@ -0,0 +1,559 @@
+Capell-1956-558-432	730	SNAKE	432	snake	Capell1956[109]
+Capell-1956-558-433	2069	SOAK (VERB)	433	soak	Capell1956[110]
+Capell-1956-558-434	1856	SOFT	434	soft	Capell1956[110]
+Capell-1956-558-435	1620	SON	435	son	Capell1956[110]
+Capell-1956-558-436	823	SORCERER	436	sorcerer	Capell1956[110]
+Capell-1956-558-437	51	SORE	437	sore	Capell1956[111]
+Capell-1956-558-438	1875	REGRET	438	sorry	Capell1956[111]
+Capell-1956-558-439	1595	SOUTH	439	south	Capell1956[111]
+Capell-1956-558-440	1623	SPEAK	440	speak	Capell1956[111]
+Capell-1956-558-441	473	SPEAR (SOMEBODY)	441	spear (vb.)	Capell1956[112]
+Capell-1956-558-442	811	SOUL	442	soul	Capell1956[112]
+Capell-1956-558-443	945	SPEAR	443	spear (n.)	Capell1956[112]
+Capell-1956-558-444			444	spear-barbed	Capell1956[112]
+Capell-1956-558-445			445	spear-fish	Capell1956[113]
+Capell-1956-558-446			446	spear-stone	Capell1956[113]
+Capell-1956-558-447			447	spear-shovel	Capell1956[113]
+Capell-1956-558-448	843	SPIDER	448	spider	Capell1956[113]
+Capell-1956-558-449	1440	SPIT	449	spit	Capell1956[114]
+Capell-1956-558-450	38	SPREAD OUT	450	spread	Capell1956[114]
+Capell-1956-558-451	849	SPRING (OF WATER)	451	spring	Capell1956[114]
+Capell-1956-558-452	1442	STAND	452	stand	Capell1956[114]
+Capell-1956-558-453	1430	STAR	453	star	Capell1956[115]
+Capell-1956-558-454	713	STEAL	454	steal	Capell1956[115]
+Capell-1956-558-455	1910	STINGRAY	455	stingray	Capell1956[115]
+Capell-1956-558-456	819	STOMACH	456	stomach	Capell1956[115]
+Capell-1956-558-457	857	STONE	457	stone	Capell1956[116]
+Capell-1956-558-458	677	AXE	458	stone axe	Capell1956[116]
+Capell-1956-558-459			459	stone headed spear	Capell1956[116]
+Capell-1956-558-460	1404	STRAIGHT	460	straight	Capell1956[116]
+Capell-1956-558-461	362	STRING	461	string	Capell1956[117]
+Capell-1956-558-462	785	STRONG	462	strong	Capell1956[117]
+Capell-1956-558-463	1518	STUPID	463	stupid	Capell1956[117]
+Capell-1956-558-464	1421	SUCK	464	suck	Capell1956[117]
+Capell-1956-558-465	1343	SUN	465	sun	Capell1956[118]
+Capell-1956-558-466	126	SWEAT (PERSPIRE)	466	sweat	Capell1956[118]
+Capell-1956-558-467	717	SWEET	467	sweet	Capell1956[118]
+Capell-1956-558-468	1439	SWIM	468	swim	Capell1956[118]
+Capell-1956-558-469	1220	TAIL	469	tail	Capell1956[119]
+Capell-1956-558-470	1749	TAKE	470	take	Capell1956[119]
+Capell-1956-558-471	2708	TALK	471	talk	Capell1956[119]
+Capell-1956-558-472	797	TESTICLES	472	testicles	Capell1956[119]
+Capell-1956-558-473	800	THIGH	473	thigh	Capell1956[120]
+Capell-1956-558-474	2308	THIN	474	thin	Capell1956[120]
+Capell-1956-558-475	2271	THINK	475	think	Capell1956[120]
+Capell-1956-558-476	847	THIRSTY	476	thirsty	Capell1956[120]
+Capell-1956-558-477	1161	THREAD	477	thread	Capell1956[121]
+Capell-1956-558-478	492	THREE	478	three	Capell1956[121]
+Capell-1956-558-479	1346	THROAT	479	throat	Capell1956[121]
+Capell-1956-558-480	1456	THROW	480	throw	Capell1956[121]
+Capell-1956-558-481	135	BOOMERANG	481	throwing-stick	Capell1956[122]
```

Comment:
A throwing-stick does not necessarily be a boomerang, or can we be sure that it definitely refers to that?